io/net/ethtool_test.py.data/ethtool_vnic_test.yaml

```diff
@@ -1,4 +1,4 @@
-interface:
+device:
```

Comment:
i like to see these changes in another commit :)

Reply:
what you mean by separate commit ?  this is just yaml change and related to py file change.. so I feel good to have in same commit... basically when there is mulitple functionalities in single commit.. than good to split ?  do you still need separate commit ?

Reply:
what I meant is if we drop the interface which is used using ` netifaces` than it will have two patch 
1- dropping support of   netifaces uses case 
2- any way this will align with actual changes